src/server.cpp

```diff
@@ -1429,7 +1429,7 @@ void Server::SendPlayerHPOrDie(PlayerSAO *playersao, const PlayerHPChangeReason
 	if (playersao->isImmortal())
 		return;
 
-	session_t peer_id   = playersao->getPeerID();
+	session_t peer_id = playersao->getPeerID();
```

Comment:
should be reverted (no other changes in this file)

Reply:
It's partially related to the PR so why not improve this tiny bit too? :man_shrugging:

Reply:
I agree with SmallJoker. I did initially touch some code in this file, which I later reverted. But I didn't revert this change, because god knows when this area of code would be touched next. ;)